CONTRIBUTING.md

```diff
@@ -279,24 +279,28 @@ With this, you should now have a working development setup! See
 
 ### Fetching the latest changes from the rustc repo
 
-Run the josh proxy locally
+The first time, register the josh proxy as a remote
 
 ```
-docker run -p 8000:8000 -e JOSH_REMOTE=https://github.com -v josh-vol:/data/git joshproject/josh-proxy:latest
+git remote add josh http://localhost:8000/rust-lang/rust.git:/src/tools/miri.git
 ```
 
-Register the josh proxy as a remote
+Each time, run the josh proxy locally (in another terminal)
 
 ```
-git remote add josh http://localhost:8000/rust-lang/rust.git:/src/tools/miri.git
+docker run -it -p 8000:8000 -e JOSH_REMOTE=https://github.com -v josh-vol:/data/git joshproject/josh-proxy:latest
 ```
 
 Fetch (takes ca 5-10 min the first time)
 
 ```
+git checkout master
+git pull
 git fetch josh
-git checkout josh/master
-git switch -c josh_sync
+git merge josh/master
+./rustup-toolchain HEAD && ./miri fmt
+git commit -am "rustup"
 ```
 
-Now push `josh_sync` to your own miri fork and open a PR with it.
+Now push to your own miri fork and open a PR with it.
+You can stop the josh proxy with Ctrl-C.
```

Comment:
I thought that didn't work

Reply:
It does now that I added `-it` to the docker flags :)